src/workerd/api/worker-rpc.c++

```diff
@@ -111,13 +111,17 @@ public:
       JSG_REQUIRE(FeatureFlags::get(js).getJsRpc(), TypeError,
           "The receiving Worker does not allow its methods to be called over RPC.");
 
+      auto& handler = KJ_REQUIRE_NONNULL(lock.getExportedHandler(entrypointName, ctx.getActor()),
+                                         "Failed to get handler to worker.");
+      auto handle = handler.self.getHandle(lock);
+
       // We will try to get the function, if we can't we'll throw an error to the client.
-      auto fn = tryGetFn(lock, ctx, methodName);
+      auto fn = tryGetFn(lock, ctx, handle, methodName);
```

Comment:
Apparently I stopped getting email notifications on my phone so this missed me, but funnily enough I wrote this exact same change locally about 2 hours ago :stuck_out_tongue:.

only nit: `tryGetFn()` doesn't need to take `IoContext` as a param anymore; I can address in a future PR. Thanks for getting this in! I am trying to verify if storage works from an rpc call but the test runner is putting up a fight.